zeppelin-web/README.md

```diff
@@ -19,19 +19,27 @@ It is recommended to install node 6.0.0+ since Zeppelin uses 6.9.1+ (see [creati
 
 All build commands are described in [package.json](./package.json)
 
-```sh
 # install required depepdencies and bower packages (only once)
-$ npm install -g yarn
-$ yarn install
+```
+$ npm install -g yarn bower npm-run-all rimraf grunt webpack karma karma-phantomjs-launcher
```

Comment:
@Leemoonsoo 
> I'm don't think 'rimraf grunt webpack karma karma-phantomjs-launcher' are really a required >package installed globally for development. Shell we remove them?
>Also shell we not use 'npm-run-all' package?
Answer - I don't know because this not my code and as far as I understand this code was developed for local development by UI developers whom should care about they local env.
I just tested process which was described in /zeppelin/zeppelin-web/README.md and found out that even after 
```
$ mvn package 
```
I was had to install rimraf grunt webpack karma karma-phantomjs-launcher globally for successful build
```
yarn build
```
So I just changed /zeppelin/zeppelin-web/README.md to real state of things.
If you disagree with this i can revert README.md because local UI developmen out of scope ZEPPELIN-1928